util/lambda_trigger/releases.go

```diff
@@ -1,18 +1,17 @@
 package main
 
 import (
+	"fmt"
+
 	hb "github.com/gulducat/hashi-bin/types"
 )
 
 // ReleasesURL The url to read the releases index from
-const ReleasesURL = "https://hc-releases-prod.s3.amazonaws.com/index.json"
+const ReleasesURL = "https://hc-releases-prod.s3.amazonaws.com"
 
 func getLatestVersion(productName string) (*hb.Version, error) {
-	index, err := hb.NewIndex(ReleasesURL)
-	if err != nil {
-		return &hb.Version{}, err
-	}
-	product, err := index.GetProduct(productName)
+	url := fmt.Sprintf("%s/%s/%s", ReleasesURL, productName, "index.json")
```

Comment:
For extra safety it might be worth calling `url.PathEscape` on `productName`. Not that any of our product names actually have slashes, but from a security perspective if someone manages to inject a weird product name they could use this to have some fun...